app/helpers/application_helper.rb

```diff
@@ -1,2 +1,13 @@
 module ApplicationHelper
-end
+    def display_author(content)
+      author_string = "<p>Author: "
+      if content.user.nil?
+        author_string += "Deleted User"
+      else
+        author_string += content.user.username
+      end
+      author_string += "</p>"
+      author_string.html_safe
```

Comment:
Dont ever use html safe if possible

Change this methods parameter to accept user rather than a mixed of comment instance or article instance to make things less prone to errors. 

About the html safe, use content_tag method

Reply:
Done